src/main/java/it/gov/pagopa/payhub/activities/dto/export/debtposition/PaidInstallmentExportFlowFileDTO.java

```diff
@@ -0,0 +1,173 @@
+package it.gov.pagopa.payhub.activities.dto.export.debtposition;
+
+import com.opencsv.bean.CsvBindByName;
+import it.gov.pagopa.payhub.activities.enums.EntityIdentifierType;
+import it.gov.pagopa.payhub.activities.enums.UniqueIdentifierType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class PaidInstallmentExportFlowFileDTO {
+    private static final String EXPORT_PAID_VERSION_V1= "v1.0";
+    private static final String EXPORT_PAID_VERSION_V1_1= "v1.1";
+    private static final String EXPORT_PAID_VERSION_V1_2= "v1.2";
+    private static final String EXPORT_PAID_VERSION_V1_3= "v1.3";
+
+    @CsvBindByName(column = "IUF", profiles = EXPORT_PAID_VERSION_V1 )
```

Comment:
you have to explicit report all versions which should write this column!
also for the other fields